processes/process_sleepspindle_parameters/process_sleepspindle_parameters

```diff
@@ -0,0 +1,202 @@
+                                     '2. Spindle frequency in Hz.<BR>'...
+                                     '3. Spindle maximum peak to peak amplitude.<BR>'...
+                                     '4. Spindle symmetry in percentage of the duration.<BR><BR>'...
+                                     'Definition of spindle parameters from "Warby et al. (2014)<BR>'...
+                                     'Nat Methods . 2014 Apr;11(4):385-92. doi: 10.1038/nmeth.2855."<BR><BR>'...
+                                     'Notes<BR>'...
+                                     'A) This process assumes that the data was already filtered between 11-16 Hz.<BR>'...
+                                     'B) This process will generate 4 files - duration, frequency, amplitude and symmetry.<BR>'...
+                                     'C) Each file x-axis = spindle number but the units will = Time(s). Cannot be changed.<BR>'...
+                                     'D) Follow the online tutorial which will take you to the GitHub README.md written by the<BR>'...
+                                     'author of this process to further understand how the process works.<BR><BR>'
+                                     ];
+    sProcess.options.info.Type    = 'label';
+    sProcess.options.info.Value   = [];
+end
+
+
+%% ===== FORMAT COMMENT =====
+function Comment = FormatComment(sProcess) %#ok<DEFNU>
+     Comment = sProcess.Comment;
+end
+
+
+%% ===== RUN =====
+function OutputFiles = Run(sProcess, sInputs) %#ok<DEFNU>
+    % Initialize returned list of files
+    OutputFiles = {};
+    Nepochs     = length(sInputs); % Numer of inputs or epochs
+    EpochN      = linspace (1,(Nepochs),(Nepochs)); % Changes the x-axis from time (s) to epoch number n. 
+    DataMat     = in_bst(sInputs(1).FileName, [], 0); % Read the first file in the list, to obtain channel size. 
+    epochSize   = size(DataMat.F);    
+    Nchannels   = epochSize(1); % Number of ALL channels (i.e. EEG, EOG, EMG and status)
+
+    % Generate matrices of zeros (Nchannels X Nepochs) to concatenate results
+    SS_Dur = zeros (Nchannels,Nepochs); % Spindle duration
+    CountP = zeros (Nchannels,Nepochs); % Count the number of peaks
+    PeakPo = zeros (Nchannels,Nepochs); % Positive peaks
+    PeakNe = zeros (Nchannels,Nepochs); % Negative peaks 
+    TimePP = zeros (Nchannels,Nepochs); % Time of max positive peak
+    TimeNP = zeros (Nchannels,Nepochs); % Time of max negative peak
+
+    % ===== LOAD THE DATA =====
+    for iNepochs = 1 : Nepochs
+       DataMat   = in_bst(sInputs(iNepochs).FileName, [], 0);
+       epochSize = size(DataMat.F);  
+       Ntime     = epochSize(2); % Number of time samples changes depending on sleep spindle
+       Data      = DataMat.F; % Actual data containing EEG, EOG and EMG data
+       Time      = DataMat.Time(end); % Time information of the recording
+       Fs        = (Ntime-1)/Time; % Calculation of Sampling frequency
```

Comment:
Safer to compute as below. E.g, in the case if negative values in the Time vector

```
Time = DataMat.Time(end) - DataMat.Time(1);
Fs   = 1 ./ (DataMat.Time(2) - DataMat.Time(1));
```

Reply:
updated as appropriate